pennylane_lightning/lightning_kokkos/lightning_kokkos.py

```diff
@@ -42,14 +42,12 @@
 from pennylane.transforms.core import TransformProgram
 from pennylane.typing import Result, ResultBatch
 
+from ._measurements import LightningMeasurements
 from ._state_vector import LightningStateVector
```

Comment:
What do you think?
```suggestion
from ._measurements import LightningKokkosMeasurements
from ._state_vector import LightningKokkosStateVector
```

Reply:
Good observation. The plan for the future is to merge all similar classes between devices but for the sake of time we will duplicate it for now :+1: